Regsitering Audit annotations (#47451)
kubernetes/website: content/en/docs/reference/labels-annotations-taints/audit-annotations.md

Example: `k8s.io/removed-release: "1.22"`

> Value **must** be in the format "\<MAJOR>\.\<MINOR>\". It is set to target the removal release

Reviewer: Why this change?

Author: When i tested the page . there was blank spaces instead of "<major>.<minor>" that's why

See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
for more information.

## apiserver.latency.k8s.io/etcd

Example: `apiserver.latency.k8s.io/etcd: "4.730661757s"`

> This annotation indiactes the measure of latency incurred inside the storage layer,

Reviewer: ```suggestion
This annotation indicates the measure of latency incurred inside the storage layer,
```

Reviewer 2: Minor nit - can be fixed in a follow-up PR. Or if @Ritikaa96 , you're willing to push the changes within this week - happy to get this merged ASAP.

> it accounts for the time it takes to send data to the etcd and get the complete response back.

Reviewer: ```suggestion
it accounts for the time it takes to send data to etcd and get a complete response back.
```

Author: Should we write etcd as `ETCD` here ?just wanted to confirm.

Reviewer: See https://etcd.io/ for capitalization

> Please note that it does not include the time incurred in admission, or validation.

Reviewer: ```suggestion
The value of this audit annotation does not include the time incurred in admission, or validation.
```

Author: noted. will make the update

## apiserver.latency.k8s.io/decode-response-object

Example: `apiserver.latency.k8s.io/decode-response-object: "450.6649ns"`

> This annotation records the decoding time of response object.

Reviewer: _This annotation records the time taken to_ … (what goes here)?

Author: This annotation records the time taken to decode the response object.

Reviewer: Is it one of these?
```suggestion
This annotation records the time taken to decode the response received from the storage layer (etcd).
```

```suggestion
This annotation records the time that elapses whilst the API server is decoding the response that
it is about to send.
```

?

Author: It's the first one . 
For reference : https://github.com/tnorlin/kubernetes/blob/851cf43a35862aeeafc4a0966f7d1e0836f675e4/staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration.go#L156

Author: ```go
This is latency incurred inside the function that takes an object returned from the underlying storage layer (etcd) and performs decoding of the response object.
```

## apiserver.latency.k8s.io/apf-queue-wait

> Example: `apiserver.latency.k8s.io/apf-queue-wait: ""`

Reviewer: Why is this set to the empty string as an example; can we come up with a better example?

Author: sure , I'll add some thing while updating other ones.

> This annotation records the time spent waiting in APF queue.

Reviewer: ```suggestion
This annotation records the time that a request spent queued due to API server priorities.
```

Author: alright. will do the change

> See [API Priority and Fairness] (/docs/concepts/cluster-administration/flow-control/)
> for more information regarding the APF feature

Reviewer: ```suggestion
See [API Priority and Fairness](/docs/concepts/cluster-administration/flow-control/) (APF)
for more information about this mechanism.
```
